package-lock.json

```diff
@@ -0,0 +1,2469 @@
+{
```

Comment:
Shouldn't `package-lock.json` be in the .gitignore?

Reply:
short answer no

Reply:
Then I 'd like to restate my concern with the double definition of the same  version configuration ...